src/Stratis.Features.FederatedPeg.Tests/CrossChainTransferStoreTests.cs

```diff
@@ -240,12 +240,12 @@ public void StoringDepositsWhenWalletBalanceInSufficientSucceedsWithSuspendStatu
 
                 Assert.Equal(2, transactions.Length);
 
-                // Transactions[0] inputs. Note that inputs are ordered by amount.
+                // Transactions[0] inputs. Note that inputs are ordered by time.
```

Comment:
I expect that this would be by BlockHeight -> Transaction ID -> Output number. Using "time" here may be a little bit misleading.